### DISCUSSION TITLE
[@types/node] 14 not include Object.fromEntries

### DISCUSSION THREAD

**Discussion by zyf0330** [DefinitelyTyped/DefinitelyTyped#60842] [Issues with a @types package]
I think @types/node 14 should support `Object.fromEntries`.
Though `Object.fromEntries` doesn't be included into esmascript until ES2019, but node 12 and 14 had already support it.

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
can be this simply patched with `ES2017.Object` added to `libs` configuration (you've said, runtime support is there)? There should be not much traction spent on v12 I think)

**Reply by zyf0330** [reply]
But `ES2017.Object` doesn't include `Object.fromEntries` method.
Yes, maybe node v12 and v14 doesn't be active now and we don't need to patch this. I just find some other cases like this, like `Promise.allSettled` which is in the ES2023 spec, but node v12 have had it.

**Comment by peterblazejewicz** [comment]
I'm really sorry, I meant: ES2019. Yes, someone can look into this, if there is a need to have this be complete.
